combat-trainer.lic

```diff
@@ -1171,6 +1181,15 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each do |name|
+      if !(Flags["ap-#{name}-active"] == true)
```

Comment:
You shouldn't need to do any assignment to Flags. Once you set it up you can check if it has triggered (`if Flags['blah']`) and then reset it. Since you're setting up a separate Flag for each buff, you shouldn't have to care about the text on which it was triggered.